src/assets/styles/article.css

```diff
@@ -0,0 +1,52 @@
+#article-body {
+	font-family: "Exo 2", sans-serif;
+	font-size: 18px;
+}
+
+@media screen and (min-width: 700px) {
+	#article-body p,
```

Comment:
A very similar effect to this can be achieved by just making `article-body` follow these styles instead of making each sub-element follow these styles. The only difference I've seen when I tried it is that blockquotes are made less wide than the div, but that can be fixed easily.

Reply:
I know it would be a lot easier but images need their own separate margins. Some images might be bigger or smaller than the paragraphs. The New York Times and Medium both make images independently styled from other elements, thus they can't be contained.